src/v2/Components/Search/Suggestions/SuggestionItem.tsx

```diff
@@ -77,9 +77,6 @@ const SuggestionItemLink = styled(RouterLink).attrs({
   min-height: 60px;
 `
 
-export const PLACEHOLDER = "Search by artist, gallery, style, theme, tag, etc."
-export const PLACEHOLDER_XS = "Search Artsy"
```

Comment:
These weren't being used anywhere besides inside the `SearchBar` component, so we deleted it from here.